cpp/build.sh

```diff
@@ -206,17 +206,10 @@ function install_yaml_cpp() {
   if [ ! -d "$YAML_CPP_BUILD_DIR" ] ; then
     echo -e "${COLOR_GREEN}[ INFO ] Building yaml-cpp ${COLOR_OFF}"
 
-    if [ "$PLATFORM" = "Linux" ]; then
-      SUDO="sudo"
-    elif [ "$PLATFORM" = "Mac" ]; then
-      SUDO=""
-    fi
-
     mkdir $YAML_CPP_BUILD_DIR
     cd $YAML_CPP_BUILD_DIR
     cmake $YAML_CPP_SRC_DIR -DYAML_BUILD_SHARED_LIBS=ON -DYAML_CPP_BUILD_TESTS=OFF -DCMAKE_CXX_FLAGS="-fPIC"
-    make
-    $SUDO make install
+    make install
```

Comment:
Without `sudo`, the `install` target may fail on linux: https://github.com/pytorch/serve/actions/runs/3528351851/jobs/5918363242#step:4:15444

```
CMake Error at cmake_install.cmake:57 (file):
  file INSTALL cannot copy file
  "/home/runner/work/serve/serve/cpp/_build/_deps/yaml-cpp-build/libyaml-cpp.so.0.7.0"
  to "/usr/local/lib/libyaml-cpp.so.0.7.0": Success.
```